Hide experimental API documentation under tab

PaulZhang12 opened the pull request:
Differential Revision: D62692308

Preview
https://docs-preview.pytorch.org/pytorch/torchrec/2396/index.html

svekars commented on docs/source/_templates/layout.html:
```diff
@@ -0,0 +1,48 @@
+{% endblock %}
+<!-- END OF LOCAL OVERRIDE -->
+
+<script script type="text/javascript">
```
It should match what you have in the toctree caption. It looks like you have only toctree captions at the moment:
'Introduction', 'All API References'. 

```suggestion
<script type="text/javascript">
```

svekars commented on docs/source/_templates/layout.html:
```diff
@@ -0,0 +1,48 @@
+{% extends "!layout.html" %}
+
+{% block extrahead %}
```
you can delete everything from line 3 to 24 and 29 to 48 - those overrides were specific to ExecuTorch unless you want your left nav to work like there.

svekars commented on docs/source/_templates/layout.html:
```diff
@@ -0,0 +1,5 @@
+{% extends "!layout.html" %}
+
```
```suggestion

{% block footer %}
{{ super() }}

```

svekars commented on docs/source/_templates/layout.html:
```diff
@@ -0,0 +1,5 @@
+
+<script type="text/javascript">
+  var collapsedSections = ['Introduction', 'All API References']
+</script>
```
```suggestion
</script>

{% endblock %}
```

PaulZhang12 committed:
Overview Docs (#2395)

Summary: Pull Request resolved: https://github.com/pytorch/torchrec/pull/2395

Differential Revision: D62691954

PaulZhang12 committed:
Hide experimental API documentation under tab (#2396)

Summary: Pull Request resolved: https://github.com/pytorch/torchrec/pull/2396

Differential Revision: D62692308
